lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -29,13 +29,15 @@
                      * @returns {HtmlSnippet} - HTML list for course-wide discussion topics.
                      */
                     getCourseWideDiscussionsHtml: function(courseWideDiscussions) {
-                        var subCategoryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
+                        var entryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
```

Comment:
The underlying tree structure of the discussion topics is bit confusing, for course wide topics entries are sub categories when we compare them with inline but on other hand they're categories too since they don't have parent.
I would go for comments or tree structure as an example that can help someone to understand the conventions.